modules/op-running-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -28,7 +28,7 @@ If the pull request author does not meet the requirements, another user who meet
 
 [discrete]
 .Pipeline run execution
-A pipeline run always runs in the namespace of the `Repository` CRD associated with the repository that generated the event.
+A pipeline run always runs in the namespace of the `Repository` custom resource definition (CRD) associated with the repository that generated the event.
```

Comment:
```suggestion
A pipeline run always runs in the namespace of the `Repository` custom resource (CR) associated with the repository that generated the event.
```

Reply:
Same here - its a part of the existing doc and no scope for review